src/mars_patcher/door_locks.py

```diff
@@ -154,6 +155,16 @@ def set_door_locks(rom: Rom, data: List[dict]) -> None:
     fix_hatch_lock_events(rom, hatch_slot_changes)
 
 
+def remove_door_palette_on_minimap(rom: Rom) -> None:
+    addresses = [0x5657A8, 0x5657C8, 0x5657E8]
+    for palette_addr in addresses:
+        palette = Palette(1, rom, palette_addr)
+        for index in range(8, 16):
+            palette.colors[index] = palette.colors[2]
+        palette.colors[10] = palette.colors[1]
+        palette.write(rom, palette_addr)
```

Comment:
1. I think remove_door_colors_on_minimap would be a better name. "Palette" usually refers to the whole row of 16 colors
2. You should add a comment explaining what the addresses are. I assume they're the palettes for downloaded, normal, and hidden
3. Color 10 is used for the red door pixels along the edge of a tile. Making it white will make red door tiles look like they have no door (see image)
4. The minimap in the top right corner while playing uses a different palette, and the colors are in a different order (see image)
5. Since the filler color (pink or green) animates on the pause screen, it looks a bit weird when the replaced door colors don't animate, but this is minor so maybe it's fine
![Screenshot 2024-09-12 112605](https://github.com/user-attachments/assets/e10c50f1-5578-4ef3-b63b-b3c896b87896)
![Screenshot 2024-09-12 112919](https://github.com/user-attachments/assets/40661fad-e44e-44db-98d0-2eaa71161006)

Reply:
3. This was a compromise that I had to make. If i don't replace color 10 with white, then save and refill stations will look very weird:
Commented:
![grafik](https://github.com/user-attachments/assets/7ae0af41-c8bb-4e78-b4ac-4047ab3ce166)
uncommented:
![grafik](https://github.com/user-attachments/assets/f64b347f-806a-474c-a15f-465f8cbeafcf)
It also doesnt seem to affect every red door, since some of the red doors in s4 show up fine
![grafik](https://github.com/user-attachments/assets/e239db0a-6708-4ec9-ade9-24f4ee2e4bb2)


4. good catch

5. yes, it looks a bit weird but its nothing major so I just left it in.